.github/workflows/nextjs.yml

```
@@ -0,0 +1,29 @@
+# This workflow runs whenever a PR is opened or updated, or a commit is pushed to main. It runs
+# several checks:
+# - check: checks that the code does not contain any biome warnings
+
+# This configuration allows maintainers of this repo to create a branch and pull request based on
+# the new branch. Restricting the push trigger to the main branch ensures that the PR only gets
+# built once.
+on:
+  push:
+    branches: [main]
+  pull_request:
```

Puedes configurar un path aqui https://docs.github.com/en/actions/writing-workflows/workflow-syntax-for-github-actions#onpushpull_requestpull_request_targetpathspaths-ignore

Para que solo se lancen estos workflows cuando haya un push a `crates/web/frontend`
```suggestion
  pull_request:
    paths: crates/web/frontend/**
```